capstone/src/main/webapp/script.js

```diff
@@ -108,6 +108,15 @@ async function loadAnnouncements () {
   }
 }
 
+async function loadCalendar () {
+  var params = new URLSearchParams(window.location.search);
+  const response = await fetch('/clubs?name=' + params.get('name'));
+  const json = await response.json();
+  const firstOfficer = json['officers'][0];
```

Comment:
In JS would this ever cause an issue if, for some reason, there are no officers ?

Reply:
Hm there shouldn't ever be no officers, but I suppose that would cause an issue - I'll add in a check for that